### ISSUE TITLE
Name and description is not imported with GPX

### ISSUE THREAD

**Issue by SemaphoreOxalis** [umap-project/umap#2058]
**Describe the bug**
Name and Description anchor seems not to be parse during import process for gpx files

**To Reproduce**
1. Tested with this file
[Cirkwi-Sentier_huguenot_-_Beaussais.zip](https://github.com/user-attachments/files/16602150/Cirkwi-Sentier_huguenot_-_Beaussais.zip)

**Expected behavior**
Having the Name and description imported.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser Firefox
 - Version 128

**Comment by SemaphoreOxalis** [comment]
Indeed. For these files, the name and description are in the metadata not inside trk because one file = one track.
I'm expecting the name and description are store in the created object (it's what you name linestring ?).
My today case is : 50 gpx/track to import in a already created layer and I want each object/track to be named. For now each new object get by default the layer name so I get 40 "Ballades" object.
Thanks.

**Comment by yohanboniface** [comment]
How did you create those GPX ?

**Comment by SemaphoreOxalis** [comment]
I download them from a map plateform : https://www.cirkwi.com/fr/recherche?lat=46.21649&lng=-0.16864&zoom=13&types=circuit;&locomotions=2;27;28;24;39;4;3;16;15;26;29;25;38;37;36;7;12;8;30;6;5;35;11;20;31;32;33;34;18;19;13;21;22;23;&categories=6;3;4;8;11;7;10

**Comment by yohanboniface** [comment]
OK thanks, so maybe that's an issue on their side ?

I'm trying to understand how much this use case (no name/descr on the trk, but on the top level metadata, and only one trk) is a common one, or an exception. As fixing this in uMap will pollute the importer code (which means more maintenance, edge cases, etc.).

**Comment by SemaphoreOxalis** [comment]
I really understand your concern and I think each case could appear because name and desc in metada is in the standard as having them in trk. The best option would be ta detect and decide : 

- a file with name and desc in metada and only one trk
- - if any name/desc in trk : use it
- - if no name/dec in trk use metada for creating the object
- a file with name and desc in metadata but multiple trk (I don't test it) : multiple object created ?
- - so using the metada for creating a layout with these info and creating object with trk name if exist or metadata if not

**Comment by pfbreton-perso** [comment]
Here is how GPX.Studio would display this file in its UI if that helps:

![image](https://github.com/user-attachments/assets/273b0ed9-2669-40f1-bfed-bd9a110518f0)
